vhdl_lang/src/syntax/disconnection.rs

```diff
@@ -0,0 +1,95 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) 2018, Olof Kraigher olof.kraigher@gmail.com
+
+use super::common::ParseResult;
+use super::expression::parse_expression;
+use super::names::parse_name;
+use super::separated_list::parse_ident_list;
+use super::tokens::{Kind::*, TokenSpan};
+use crate::ast::token_range::WithTokenSpan;
+use crate::ast::*;
+use vhdl_lang::syntax::parser::ParsingContext;
+
+/// LRM 7.4 Disconnection Specification
+pub fn parse_disconnection_specification(
+    ctx: &mut ParsingContext<'_>,
+) -> ParseResult<WithTokenSpan<DisconnectionSpecification>> {
+    let start_token = ctx.stream.expect_kind(Disconnect)?;
+    if ctx.stream.next_kind_is(Identifier) {
+        let _ = parse_ident_list(ctx);
```

Comment:
As I said in a previous comment, this information should be kept in the `DisconnectSpecification` struct.

I really like the separation of concerns in this file though, i.e., that you created a separate file for the purposes of parsing the disconnect statement.

Reply:
Done, see comment above.